lib/tasks/docker.rake

```diff
@@ -1,6 +1,8 @@
 require "#{File.dirname(__FILE__)}/../../lib/slide_hub/version"
 require 'open3'
 
+BASE_CONTAINER_VERSION = '20180314'
```

Comment:
__[RuboCop]__

Style/MutableConstant: Freeze mutable objects assigned to constants.


<!-- Comment by [SideCI](https://sideci.com) -->